prov/efa/src/rxr/rxr_ep.c

```diff
@@ -743,11 +743,8 @@ static int rxr_ep_close(struct fid *fid)
 		retv = ret;
 	}
 
-	ret = fi_close(&rxr_ep->rdm_cq->fid);
-	if (ret) {
-		FI_WARN(&rxr_prov, FI_LOG_EP_CTRL, "Unable to close msg CQ\n");
-		retv = ret;
-	}
+	struct ibv_cq *ibv_cq = ibv_cq_ex_to_cq(rxr_ep->ibv_cq_ex);
+	ret = -ibv_destroy_cq(ibv_cq);
```

Comment:
check `ret` and print warning if it is not 0.

Reply:
will fix